src/web/app/pages-instructor/instructor-session-base-page.component.ts

```diff
@@ -163,6 +164,8 @@ export abstract class InstructorSessionBasePageComponent {
       model.instructorPrivilege = instructorPrivilege;
     }, (resp: ErrorMessageOutput) => {
       this.statusMessageService.showErrorToast(resp.error.message);
+    }, () => {
```

Comment:
might want to consider using `pipe(finalize(() => model.isLoadingInstructorPrivilege = false))` instead. Otherwise if the subscription errors the complete clause is never called.

Reply:
Ah okay I didn't know that, thank you!